atd-toolbox/data_model/python/column_metadata.json

```diff
@@ -359,7 +359,7 @@
   {
     "column_name": "address_primary",
     "data_type": "text",
-    "data_type_constraint": "generated always as (trim(coalesce(rpt_street_pfx, '') || ' ' || coalesce(rpt_block_num, '') || ' ' || coalesce(rpt_street_name, '') || ' ' || coalesce(rpt_street_sfx, ''))) stored",
+    "data_type_constraint": "generated always as (build_crash_address(rpt_block_num, rpt_street_pfx, rpt_street_name, rpt_street_sfx)) stored",
```

Comment:
subtle change, but i've also changed the concatenation so that `rpt_block_num` comes before `rpt_street_prx` — this is a deviation from VZE prod that Xavier requested. 

So `N 6400 LAMAR BLVD` becomes `6400 N LAMAR BLVD` 👍